controller/cache/cluster.go

```diff
@@ -315,10 +315,21 @@ func (c *clusterInfo) ensureSynced() error {
 	return c.syncError
 }
 
-func (c *clusterInfo) iterateHierarchy(obj *unstructured.Unstructured, action func(child appv1.ResourceNode)) {
+func (c *clusterInfo) getNamespaceTopLevelResources(namespace string) map[kube.ResourceKey]appv1.ResourceNode {
```

Comment:
no test coverage?

Reply:
how about naming func as `getUnownedResources`

Reply:
The method returns namespace resources which have no parents ( top level ), not unknown resources.